### ISSUE TITLE
Move items to bank, not reagents

### ISSUE THREAD

**Issue by akbyrd** [arkayenro/arkinventory#1562]
**What version number you are using?**
3.09.63

**What game client are you playing? Retail, Burning Crusade, Classic / Live, PTR, Beta**
Retail

**What language is the game client set to?**
English

**Ask your question**
When I deposit items into my bank by right clicking them from my bags ArkInventory automatically moves some things to the Reagents bag. I don't want this behavior. I'd like items to go to whichever bag is currently open, like they normally do. How do I disable this feature?

**Comment by arkayenro** [comment]
right click on the restack icon and select the options you want to turn on/off

**Comment by akbyrd** [comment]
In my regular bag or my bank bag? I don't see any options that affect this feature.
![image](https://user-images.githubusercontent.com/484784/158913891-c8ce1e77-6049-43dc-af22-00827edfd4e4.png)
![image](https://user-images.githubusercontent.com/484784/158913907-df01fa42-ce25-49d4-a201-8eb2d277607e.png)

**Comment by arkayenro** [comment]
if youre using the blizzard cleanup then i dont think that has any options, it just does it by default.

you can either shift to arkinventory restack and play with the options for that to see if it works the way you want.

**Comment by akbyrd** [comment]
Is ArkInventory automatically running cleanup/restack when depositing items? I think that's the behavior I want to disable.

**Comment by arkayenro** [comment]
i dont think so, if you do a deposit all reagents it should only do that, nothing else
